packages/s3-replicator/test/test-index.js

```diff
@@ -0,0 +1,13 @@
+const test = require('ava');
+
+const { handler } = require('../index');
+
+test('handler returns immediately on non-create event', async (t) => {
```

Comment:
Wouldn’t mind some feedback on how to write a good unit test for the do-copy case, considering it’s really just a single `copyObject` call.

Reply:
I don't have a good test except to test the implementation, which isn't to useful.